test/python/basicaer/test_statevector_simulator.py

```diff
@@ -45,9 +55,6 @@ def test_run_circuit(self):
 
     def test_measure_collapse(self):
         """Test final measurement collapses statevector"""
```

Comment:
The circuit setting seems missing...
```suggestion
        """Test final measurement collapses statevector"""
        bell = QuantumCircuit(2)
        bell.h(0)
        bell.cx(0, 1)
        self.circuit = bell
```